packages/pwa-kit-dev/package.json

```diff
@@ -103,6 +104,7 @@
     "source-map-loader": "^4.0.1",
     "speed-measure-webpack-plugin": "^1.5.0",
     "svg-sprite-loader": "^6.0.11",
+    "tsx": "^4.16.2",
```

Comment:
https://github.com/privatenumber/tsx

TODO: 3PP

Reply:
This package adds runtime support for importing typescript or es module files.

Reply:
3pp submitted, it's been approved in dozens of previous versions, so i'm confident this package should have no trouble with 3pp